Content.Server/GameObjects/Components/Kitchen/GrindableComponent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.GameObjects.Components.Kitchen
+{
+
+    /// <summary>
+    /// Tag component that denotes an entity as Grindable by the reagentgrinder.
+    /// </summary>
+    [RegisterComponent]
+
+    public class GrindableComponent : Component
```

Comment:
Should this not be like JuiceableComponent and have an output entity/something as a variable?